examples/triton/docker-compose-multi-datacenter.yml.template

```diff
@@ -0,0 +1,23 @@
+version: '2.1'
```

Comment:
I'd rather have this file within `examples/multi-dc/docker-compose.yml.tmpl` and not under `examples/triton` so that users can understand how to run Consul on Triton without getting DC involved. `examples/multi-dc` can also host all the templates you are rendering.